pkg/reconciler/model.go

```diff
@@ -37,11 +42,20 @@ type Reconciliation struct {
 	CallbackURL     string          `json:"callbackURL"` //CallbackURL is mandatory when component-reconciler runs in separate process
 	InstallCRD      bool            `json:"installCRD"`
 	CorrelationID   string          `json:"correlationID"`
+	Repo            Repo            `json:"repo"`
```

Comment:
Can we use "repository" in the JSON as the other fields were also avoiding abbreviations?